docs/5.0/pages/installation.md

```diff
@@ -93,13 +58,24 @@ up-to-date information.
     $ tar -xzf teleport-v{{ teleport.version }}-linux-arm64-bin.tar.gz
     $ cd teleport
     $ ./install
-    $ which teleport
-    /usr/local/bin/teleport
+    ```
+
+=== "Tarball"
+
+    ```bash
+    $ curl https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ cd teleport
+    $ ./install
     ```
 
 ## Docker
 
-Please follow our [OSS Docker Quickstart](quickstart-docker.md) or [Enterprise Docker Quickstart](enterprise/quickstart-enterprise.md#run-teleport-enterprise-using-docker) for install and setup instructions.
+Please follow our [OSS Docker Quick Start](quickstart-docker.md) or [Enterprise Docker Quick Start](enterprise/quickstart-enterprise.md#run-teleport-enterprise-using-docker) for install and setup instructions.
```

Comment:
Let's use `Getting started` as a phrase, see https://goteleport.com/teleport/docs/docs

Reply:
Updated these links and modified the language on the quickstart page.